### PR TITLE
Cut releases for Node 8.5

### PR DESCRIPTION
**Pull request by amesgen**
Also inlcudes changelog entries for #366 and the first commit in #323 (second commit was reverted in #362).

### CODE REVIEW

File: ouroboros-consensus-protocol/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 # Changelog entries
 
+<a id='changelog-0.5.0.7'></a>
+## 0.5.0.7 — 2023-09-27
+
+### Non-Breaking
```
Comment (nfrisby) [inline ouroboros-consensus-protocol/CHANGELOG.md:8]: Isn't this a `Patch` change, not `Non-Breaking`?

File: ouroboros-consensus-diffusion/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 # Changelog entries
 
+<a id='changelog-0.8.0.1'></a>
+## 0.8.0.1 — 2023-09-27
+
+### Non-Breaking
```
Comment (nfrisby) [inline ouroboros-consensus-diffusion/CHANGELOG.md:8]: Also, `Patch`, right? Just based on the version number you chose.

Or maybe the version bump is too gentle.

**Review by nfrisby — APPROVED** [review]
Approved. Disclaimer:

- I pointed out that the changelog labels and the actual version changes seem incoherent (patch versus non-breaking), but I trust you to resolve that without another round of review.

- I see that some of the changelog content here is not arriving via `scriv`. I did not check whether what you wrote covers _all_ PRs that were missing their changelog fragments.

File: ouroboros-consensus-protocol/CHANGELOG.md (hunk 1 above)
Comment (amesgen) [reply ouroboros-consensus-protocol/CHANGELOG.md:8]: Ah thanks, this should indeed be `Patch`, will change :+1:

File: ouroboros-consensus-diffusion/CHANGELOG.md (hunk 2 above)
Comment (amesgen) [reply ouroboros-consensus-diffusion/CHANGELOG.md:8]: https://github.com/input-output-hk/ouroboros-consensus/pull/377#discussion_r1339685322

**Comment by amesgen** [comment]
> I see that some of the changelog content here is not arriving via `scriv`. I did not check whether what you wrote covers *all* PRs that were missing their changelog fragments.

Thanks for pointing that out, I updated the PR description to mention that.

**Pull request merged**